dmscripts/generate_framework_agreement_signature_pages.py

```diff
@@ -51,6 +53,7 @@ def save_page(html, supplier_id, output_dir, descriptive_filename_part):
     page_path = os.path.join(output_dir, '{}-{}.html'.format(supplier_id, descriptive_filename_part))
     with io.open(page_path, 'w+', encoding='UTF-8') as htmlfile:
         htmlfile.write(html)
+    return page_path
 
 
 def render_html_for_successful_suppliers(rows, framework, template_dir, output_dir, dry_run=False):
```

Comment:
Don't think so? Did you have anything specific in mind?

Reply:
I was just wondering because it makes use of lines 60 and 61 that use the template `framework-agreement-signature-page.html` and css, which, if I understand correctly, is the pre e-signature template? I may be completely way off, so I am sorry if I am wrong.

Reply:
@phantas good question, this was also confusing for me. I think this method was used for the previous manual signature flow. i.e. to generate the PDF that was printed and signed by the supplier.

As a follow-up. Once DOS-5 goes out we should also have a card to remove all this legacy functionality.